test/token/ERC721/ERC721.behavior.js

```diff
@@ -9,17 +9,10 @@ const { RevertType } = require('../../helpers/enums');
 const firstTokenId = 5042n;
 const secondTokenId = 79217n;
 const nonExistentTokenId = 13n;
-const fourthTokenId = 4n;
-const baseURI = 'https://api.example.com/v1/';
 
 const RECEIVER_MAGIC_VALUE = '0x150b7a02';
 
-function shouldBehaveLikeERC721() {
-  beforeEach(async function () {
-    const [owner, newOwner, approved, operator, other] = this.accounts;
-    Object.assign(this, { owner, newOwner, approved, operator, other });
-  });
```

Comment:
I'm not convinced about this change. There's a transitive dependency between each test suite using `shouldBehaveLikeERC721` and their fixtures. 

In other words, the fixture of (let's say) ERC721Wrapper will need to know the values needed in `this` context (owner, newOwner, etc.). I think it's easy to lose track of unnecessary variables in fixtures this way.

Let's stick to only passing `accounts` so that this function can figure its own variables.